node/network/bridge/src/lib.rs

```diff
@@ -16,6 +16,11 @@
 
 //! The Network Bridge Subsystem - protocol multiplexer for Polkadot.
 
+#![deny(unused_crate_dependencies, unused_results)]
+#![warn(missing_docs)]
+
+use streamunordered as _;
```

Comment:
do we actually need it?

Reply:
Hmm, no actually we don't, I guess that waws an overlaying error.